src/app/nav/header-types.ts

```diff
@@ -3,6 +3,7 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
+import "clr-icons/shapes/essential-shapes";
```

Comment:
This path should be relative to the actual directory

Reply:
Does it need to be relative? Because we have several component files that already use this type of non-relative module imports in our demo app. Also, Isn't it actually better to have non-relative module imports if we ever need to change the folder structures of our project in the future?

Reply:
Even if we move folders, paths could still be wrong. However, I ran into an issue with these types of paths when working on https://github.com/vmware/clarity/pull/2015. I worked on this further, and it does not actually cause a problem now. I am wondering if I changed something with TypeScript (as I was tinkering with the solution above) that broke this. 

@jeeyun it appears I was wrong, and you could revert that change for relative path. Sorry :(

Reply:
Haha no worries. Will revert and also rebase since this is close to being merged.